src/lib.rs

```diff
@@ -432,6 +432,44 @@ impl Transform {
         Transform::new(1.0, 0.0, 0.0, 1.0, tx, ty)
     }
 
+    /// Creates a new rotation transform.
+    #[inline]
+    pub fn new_rotate(angle: f32) -> Self {
+        #[cfg(feature = "no-std")]
```

Comment:
I would suggest going the `tine-skia` route by using [a trait](https://github.com/RazrFalcon/tiny-skia/blob/784569522f728cc70232cbb88abd7330571b51dd/path/src/scalar.rs#L78-L125).

And then 
```rust
#[cfg(all(not(feature = "std"), feature = "no-std-float"))]
use crate::NoStdFloat;
```

where needed.

Reply:
You can put the new trait somewhere in `lib.rs`